### ISSUE TITLE
My Collection not retaining selections

### ISSUE THREAD

**Issue by AceDashDog** [raithos/xwing#1220]
Logged in under Google I select the expansions, core box and odd ships I own in My Collection. Having Closed the My Collection window, I close YASB. When I come back, the selections are gone. In one case one remained while the others disappeared

**To Reproduce**
See above

**Expected behavior**
My Collection retains my selections

**Screenshots**

**Desktop (please complete the following information):**
Windows 10 Pro desktop
Windows 7 Pro Laptop
Windows 10 Pro Laptop
Updated Chrome Browser in all cases

Love the app none the less :)
Cheers!

**Comment by raithos** [comment]
i just tested it, and my collections are being saved. you sure youre using yasb.app and not anything else?

**Comment by AceDashDog** [comment]
Yes, I am. I did experiment with having two logins, one Google one Discord, so I could tally up a non-techie friend's collection. 
 
Oddly it seems to now be working. After 30 years in IT support, the thing I hated most (after fax machines) was an intermittent fault! 

I'll come back if it happens again. Thanks for the response - and thanks for the app, it's most excellent 
 

--



Hugh Foster 

***@***.***

http://www.ace-dog.com 

I've had a perfectly wonderful evening. But this wasn't it. (Groucho Marx)

--



Wednesday, January 11, 2023, 8:01:20 AM, you wrote:
